functions/exercise4/scriptSpect.js

```diff
@@ -0,0 +1,8 @@
+/// <reference path="../../typings/globals/jasmine/index.d.ts" />
+/*globals validatePhrase*/
+
+describe("Check if", function () {
+  it("phrase is number ", function () {
+    expect(function () { validatePhrase(Number); }).toThrow(new Error("Phrase can't be number"));
```

Comment:
Why do you provide Number function:
https://www.w3schools.com/jsref/jsref_number.asp
here as a parameter?
Should your function get function as a parameter?